marklogic-data-hub/src/server-side/impl/trace-lib.xqy

```diff
@@ -628,8 +628,7 @@ declare function trace:get-trace($id as xs:string)
 {
   let $query :=
     cts:or-query((
-      cts:element-range-query(xs:QName("traceId"), "=", $id, ("collation=http://marklogic.com/collation/codepoint")),
-      cts:json-property-range-query("traceId", "=", $id, ("collation=http://marklogic.com/collation/codepoint"))
+      cts:path-range-query("/trace/traceId", "=", $id, ("collation=http://marklogic.com/collation/codepoint"))
```

Comment:
has the traceDB config been updated to add this path range index?

Reply:
never had this other index.